### DISCUSSION TITLE
RelationManager under RelationManager

### DISCUSSION THREAD

**Discussion by ramhanya** [filamentphp/filament#11379] [Help] [answered]
### Package

Other

### Package Version

v3

### How can we help you?

I'm building admin and user panel with filament.
in admin panel i have a usersresource and relationManger that is name is "CampaignsRelationManager".
in user panel i have a campaigns resource and relationManger that is name is "CampCouponRelationManager".

**i want** to add CampCouponRelationManager to CampaignsRelationManager under the admin panel.
Basically, it means to create a relationManger inside a relationManger

**Comment by ramhanya** [comment]
?

**Comment by leandrocfe** [comment, marked as answer]
https://filamentphp.com/plugins/guava-nested-resources

**Reply by ramhanya** [reply]
Thank you so much!
